src/pybind/mgr/dashboard/frontend/src/app/shared/components/motd/motd.component.scss

```diff
@@ -0,0 +1,3 @@
+::ng-deep cd-motd .no-margin-bottom {
+    margin-bottom: 0;
```

Comment:
The indention is wrong, please run `npm run fix` to fix that.

Reply:
Done